elytron/WFLY-11868_RESTEasy_client_integration_with_Elytron.adoc

```diff
@@ -0,0 +1,86 @@
+= WFLY-11868 REST integration with WildFly Elytron - AuthenticationClient for Authentication / SSL
+:author:            Diana Vilkolakova
+:email:             dvilkola@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          elytron, client, resteasy
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+Wildfly-config.xml file is used by Elytron client and it contains the authentication configuration that should be used
+when attempting to connect to the server (username, password, ssl config, mechanism specification etc).
+This requested feature is about making use of Elytron and wildfly-config.xml file on the client side of REST deployment.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/WFLY-11697[WFLY-11868]
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/EAP7-1097[EAP7-1219]
```

Comment:
Please fix the link, right one is likely https://issues.jboss.org/browse/EAP7-1219